backend/src/node/connector.rs

```diff
@@ -71,7 +81,9 @@ impl NodeConnector {
     }
 
     fn handle_message(&mut self, msg: NodeMessage, data: Bytes, ctx: &mut <Self as Actor>::Context) {
-        if let Some(chain) = self.chain.as_ref() {
+        let conn_id = msg.id.unwrap_or(0);
```

Comment:
I take it `0` lets us be backwards compatible?

Reply:
Yup yup